classes/event/entry_created.php

```diff
@@ -42,4 +44,18 @@ public function init() {
         $this->data['objecttable'] = 'cpdlogbook_entries';
     }
 
+    public function get_description() {
+        $a = new \stdClass();
+        $a->userid = $this->get_data()['userid'];
+        $a->entryid = $this->get_data()['objectid'];
+        $a->name = $this->get_data()['other']['entryname'];
+
+        return "The user with id '{$a->userid}' created the '{$a->name}' cpdlogbook entry with id '{$a->entryid}'.";
+    }
+
+    public function get_url() {
+        $id = $this->get_data()['objectid'];
+        return new moodle_url('/mod/cpdlogbook/edit.php', ['id' => $id, 'create' => false]);
```

Comment:
Ah I've just realized we should have a 'detail' view page for a single entry and link to that rather than an edit page. I'll merge this and we can fix that later in another card